backend/src/apiserver/server/pipeline_server.go

```diff
@@ -123,7 +124,18 @@ func (s *PipelineServer) CreatePipeline(ctx context.Context, request *api.Create
 		return nil, util.Wrap(err, "Invalid pipeline name.")
 	}
 
-	pipeline, err := s.resourceManager.CreatePipeline(pipelineName, request.Pipeline.Description, pipelineFile)
+	resourceReferences := request.Pipeline.GetResourceReferences()
+	namespace := common.GetNamespaceFromAPIResourceReferences(resourceReferences)
+	resourceAttributes := &authorizationv1.ResourceAttributes{
+		Namespace: namespace,
+		Verb:      common.RbacResourceVerbList,
+	}
```

Comment:
I think these aren't the correct resource attributes. In this case, we should use the `Create` verb and the `Pipelines` resource. Also, shouldn' we separate the case of namespaced and non-namespaced pipeline?